libdevcore/Common.h

```diff
@@ -294,6 +294,12 @@ enum class WithExisting: int
 	Kill
 };
 
+/// Get the current time in seconds since the epoch in UTC
+inline time_t getNowUTC()
```

Comment:
Can you make it returning a more familiar type like uint64_t? And move the implementation to cpp file?